packages/client/src/index.js

```diff
@@ -3,10 +3,14 @@ import ReactDOM from 'react-dom'
 import './shared/styles/index.css'
 import Routes from './root-routes'
 import * as serviceWorker from './serviceWorker'
+import { PlanetProvider } from './pages'
 
 ReactDOM.render(
   <React.StrictMode>
-    <Routes />
+    <PlanetProvider>
```

Comment:
If you wanted to keep the context closer to the app we could have "Page related routes" within `pages` Thats where I would usually put nested routes anyway

I.E
/pages/planet/index.js

```
<PlanetProvider>
<Switch>
    <Route path="/">
         <Planet />
    </Route>
    <Route path="/something/nested">
         <PlanetNested />
    </Route>
</Switch>
</PlanetProvider>
```

Reply:
Yes I agree. Lets make this change